src/cpp/core/include/core/r_util/RTokenizer.hpp

```diff
@@ -442,6 +442,27 @@ inline bool isWhitespaceOrComment(const RToken& rToken)
           rToken.isType(RToken::COMMENT);
 }
 
+inline bool isRoxygenComment(const RToken& rToken)
+{
+   if (!rToken.isType(RToken::COMMENT))
+      return false;
+
+   std::string content = rToken.contentAsUtf8();
+   char c;
+   for (size_t i = 1; i < content.size(); i++) 
```

Comment:
Should this start at `0` rather than `1`?

Reply:
updated, I think the original thought was that since it's a comment we already know about `i = 0`

Reply:
Ah, that makes sense. Thanks!